package.json

```diff
@@ -232,12 +232,12 @@
                 },
                 {
                     "command": "staticWebApps.viewProperties",
-                    "when": "view == staticWebApps && viewItem =~ /azureStatic(WebApp|Environment|AppSettings)$/",
+                    "when": "view == staticWebApps && viewItem =~ /azureStatic(WebApp|Environment)|applicationSettings$/",
                     "group": "9@1"
                 },
                 {
                     "command": "staticWebApps.openInPortal",
-                    "when": "view == staticWebApps && viewItem =~ /azureStatic(WebApp|AppSettings|Functions)$/",
+                    "when": "view == staticWebApps && viewItem =~ /azureStatic(WebApp|Functions)|applicationSettings$/",
```

Comment:
If you're going for consistency with the previous stuff, I think you need the `$` in both alternatives: `/azureStatic(WebApp|Functions)$|applicationSettings$/`

That being said, not sure it was actually needed in the first place